### ISSUE TITLE
Update to version 67

### ISSUE THREAD

**Issue by Eloston** [Eloston/ungoogled-chromium#414]
This issue tracks the upgrade progress to Chromium 67, subsequent revisions, and related discussion.

Here are the steps necessary towards the first release in the upgrade process:
* [x] Update `common` and other abstract base bundles
* [ ] Update other base bundles and associated packaging types:
    * [ ] Arch Linux
    * [x] Debian and Ubuntu
    * [x] macOS
    * [x] openSUSE
    * [x] Portable Linux
    * [x] Windows

NOTE: A base bundle is considered "updated" once its current configuration has been added/removed/changed to produce working builds (i.e. runs without severe regressions) on the new version. It does **not** always consider new Chromium features.

After the initial release is published, this issue will remain open for status updates and discussions relating to new revisions or the version as a whole. It will be closed once a new Specific problems should still be posted in a separate issue.

Miscellaneous TODO:
* [x] build target 'widevinecdmadapter' was removed - References to it and the resulting library need to be removed from the packaging scripts
* [x] binary pruning and domain substitution lists need to be updated when the stable version is released

**Comment by Eloston** [comment]
1b9bc64c465543101f7944aee7e8a52087d40344 can build Debian stretch packages that can't play any media. I need to refresh Portable Linux patches and try building.

**Comment by Eloston** [comment]
**Status update**: Portable Linux seems to be fully functional as of fb6a3a450508ba633914aa6a2924aeb42a620e64 with [a build available here](https://github.com/Eloston/ungoogled-chromium-binaries/releases/tag/67.0.3396.62-1)

**Comment by LeFroid** [comment]
The OpenSUSE build is also having the issue where media cannot be played.

**Comment by tectiv3** [comment]
hey, I am building for macOS atm and this line https://chromium.googlesource.com/chromium/src/base/+/master/mac/sdk_forward_declarations.h#288
does not work on 10.11 
```
error: expected ';' after top level declarator
typedef NSString* VNImageOption NS_STRING_ENUM;
```

Got to replace it with ```typedef NSString* VNImageOption;```
Should be fine on newer systems though.
I am hesitant to creating a patch for this since it looks like an upstream problem and it affects only people on older systems.

**Comment by xsmile** [comment]
Does media playback with the faulty builds produce any logs in the console or on the chrome:gpu, chrome:media-internals pages?

**Comment by Eloston** [comment]
@xsmile I didn't save the logs, but both stderr and `chrome://media-internals` said something about FFmpegDemuxer not being able to open a context (I tried looking it up, and it's a generic failure trying to load FFmpeg). When I tried opus audio (e.g. YouTube or opus audio files), it just says the audio decoder wasn't able to initialize, or something to that effect. `chrome://gpu` didn't have anything IIRC, and toggling VA-API acceleration has no effect.

I suspect this may be a FFmpeg incompatibility (not too sure about opus though). I'm just going to wait and see if Debian will push a patch to fix this or not, because I haven't seen anyone else building 67 with old FFmpeg yet.

**Comment by tectiv3** [comment]
@Eloston please merge it only after someone on 10.13 tries it first.

**Comment by nixballs** [comment]
@tectiv3 , @Eloston 

macOS 10.13.5 - latest build  - [67.0.3396.62-1](https://github.com/tectiv3/ungoogled-chromium-binaries/releases/tag/67.0.3396.62-1) causes high CPU usage, details below.
![image](https://user-images.githubusercontent.com/7742499/40975598-d88e6c7a-690e-11e8-9dbc-b2c6e4ded3f1.png)

![image](https://user-images.githubusercontent.com/7742499/40975623-e321afe4-690e-11e8-9933-1821ae35b52a.png)

**Comment by tectiv3** [comment]
yeah, I noticed this as well. Sometimes one of the tabs starts hogging CPU.

**Comment by Eloston** [comment]
Debian Stretch binaries for 67.0.3396.87-1 [have been uploaded for testing](https://github.com/Eloston/ungoogled-chromium-binaries/releases/tag/67.0.3396.87-1).

**Comment by Wyse-** [comment]
Building version 67.0.3396.87-1 on Windows seems to fail. Actually the build process doesn't even start.
This is the output I get when running build.bat:
https://paste.debian.net/plain/1030109

Also note that on same setup the build process starts normally for version 66.0.3359.181-1, so I'm guessing this is something inherent to the latest update.

**Comment by nixballs** [comment]
@tectiv3 // @Eloston 

Latest release seems to be working fine. I put the macOS binaries here [67.0.3396.87-1](https://github.com/nixballs/ungoogled-chromium/releases/tag/67.0.3396.87-1)

**Comment by tectiv3** [comment]
@nixballs no random high CPU usage? It happens once or twice a day for me on my build.

**Comment by nixballs** [comment]
@tectiv3 nope, none so far. @bjrnio also mentioned .79 was supposed to fix the high CPU. Using around 4~5 tabs and CPU is hovering around 5-10%.

**Comment by tectiv3** [comment]
It wasn't about tabs count, some random tab was consuming 100% CPU. Happened to me only couple of times in those two weeks. Good to know .79 address that!

**Comment by Eloston** [comment]
**Status update**: Windows support has been updated. [A build is available here](https://github.com/Eloston/ungoogled-chromium/issues/428#issuecomment-399771857). Please test and report any bugs; I will tag a new release after several days if there aren't any major issues.

**Comment by tectiv3** [comment]
Latest macOS build (.87) - no issues in four days.

**Comment by Wyse-** [comment]
No issues so far on the Windows build mentioned above.

**Comment by waqas** [comment]
Been using 67 on Windows last few days. No issues so far. Thanks @Eloston, @Wyse-

**Comment by Ti-R** [comment]
Hello,

I'm trying to compile this revision (The master)
https://github.com/Eloston/ungoogled-chromium/commit/7c9d189ff0c4ba33d923bc67f47998d4fb6460a2

This version possess one more "commit" to enhance the tag version
https://github.com/Eloston/ungoogled-chromium/tree/67.0.3396.87-2

All is going well until it do not find "kDisableWebRtcHWDecoding"
This var is only available if ENABLE_WEBRTC and OS_CHROMEOS are enable inside "content\public\common\content_switches.cc"

ENABLE_WEBRTC flag seems to be part of media_buildflags inside "media\BUILD.gn" file.
So the build seems to miss a flag, maybe ENABLE_WEBRTC  or OS_CHROMEOS, how can I solve the compilation part ?

Or did I not install the good SDK
My softwares are: 
Visual studio Community 2017 - 15.5.27130.2010
Windows Software Development Kit - Windows 10.0.15063.468 is install among others.

Thx to looking at my problem and point me in the good direction.

Project is great !

The error:
```
[17510/21319] CXX obj/content/browser/browser/browser_jumbo_11.obj
FAILED: obj/content/browser/browser/browser_jumbo_11.obj
../../third_party/llvm-build/Release+Asserts/bin/clang-cl.exe /nologo /showIncludes  "-imsvcC:\Program Files (x86)\Microsoft Visual Studio\2017\Community\VC\Tools\MSVC\14.12.25827\ATLMFC\include" "-imsvcC:\Program Files (x86)\Microsoft Visual Studio\2017\Community\VC\Tools\MSVC\14.12.25827\include" "-imsvcC:\Program Files (x86)\Windows Kits\NETFXSDK\4.6.1\include\um" "-imsvcC:\Program Files (x86)\Windows Kits\10\include\10.0.15063.0\ucrt" "-imsvcC:\Program Files (x86)\Windows Kits\10\include\10.0.15063.0\shared" "-imsvcC:\Program Files (x86)\Windows Kits\10\include\10.0.15063.0\um" "-imsvcC:\Program Files (x86)\Windows Kits\10\include\10.0.15063.0\winrt" "-imsvcC:\Program Files (x86)\Microsoft Visual Studio\2017\Community\VC\Tools\MSVC\14.12.25827\ATLMFC\include" "-imsvcC:\Program Files (x86)\Microsoft Visual Studio\2017\Community\VC\Tools\MSVC\14.12.25827\include" "-imsvcC:\Program Files (x86)\Windows Kits\NETFXSDK\4.6.1\include\um" "-imsvcC:\Program Files (x86)\Windows Kits\10\include\10.0.16299.0\ucrt" "-imsvcC:\Program Files (x86)\Windows Kits\10\include\10.0.16299.0\shared" "-imsvcC:\Program Files (x86)\Windows Kits\10\include\10.0.16299.0\um" "-imsvcC:\Program Files (x86)\Windows Kits\10\include\10.0.16299.0\winrt" -DENABLE_SCREEN_CAPTURE=1 -D__ATLHOST_H__ -DV8_DEPRECATION_WARNINGS -DUSE_AURA=1 -DNO_TCMALLOC -DOFFICIAL_BUILD -DCHROMIUM_BUILD -D"CR_CLANG_REVISION=\"328716-2\"" -D_HAS_EXCEPTIONS=0 -D__STD_C -D_CRT_RAND_S -D_CRT_SECURE_NO_DEPRECATE -D_SCL_SECURE_NO_DEPRECATE -DDEPRECATEDENUMERATOR(x)=[[deprecated(x)]] -D_ATL_NO_OPENGL -D_WINDOWS -DCERT_CHAIN_PARA_HAS_EXTRA_FIELDS -DPSAPI_VERSION=1 -DWIN32 -D_SECURE_ATL -D_USING_V110_SDK71_ -DWINAPI_FAMILY=WINAPI_FAMILY_DESKTOP_APP -DWIN32_LEAN_AND_MEAN -DNOMINMAX -D_UNICODE -DUNICODE -DNTDDI_VERSION=0x0A000002 -D_WIN32_WINNT=0x0A00 -DWINVER=0x0A00 -DNDEBUG -DNVALGRIND -DDYNAMIC_ANNOTATIONS_ENABLED=0 -DCONTENT_IMPLEMENTATION -DV8_USE_EXTERNAL_STARTUP_DATA -DWEBP_EXTERN=extern -DUSE_EGL -DGOOGLE_PROTOBUF_NO_RTTI -DGOOGLE_PROTOBUF_NO_STATIC_INITIALIZER -DU_USING_ICU_NAMESPACE=0 -DU_ENABLE_DYLOAD=0 -DU_STATIC_IMPLEMENTATION -DICU_UTIL_DATA_IMPL=ICU_UTIL_DATA_FILE -DUCHAR_TYPE=wchar_t -DSK_IGNORE_LINEONLY_AA_CONVEX_PATH_OPTS -DSK_HAS_PNG_LIBRARY -DSK_HAS_WEBP_LIBRARY -DSK_HAS_JPEG_LIBRARY -DSK_SUPPORT_GPU=1 -DGR_GL_FUNCTION_TYPE=__stdcall -DLEVELDB_PLATFORM_CHROMIUM=1 -DDeleteFile=DeleteFileW -DOS_WIN -DWEBRTC_NON_STATIC_TRACE_EVENT_HANDLERS=0 -DGTEST_RELATIVE_PATH -DWEBRTC_CHROMIUM_BUILD -DWEBRTC_WIN -D_CRT_SECURE_NO_WARNINGS -DMESA_EGL_NO_X11_HEADERS -DXML_STATIC -DNO_MAIN_THREAD_WRAPPING -DFLAC__NO_DLL -I. -I../.. -Igen -I../../third_party/libwebp/src -I../../third_party/khronos -I../../gpu -I../../third_party/wtl/include -I../../third_party/libyuv/include -I../../third_party/protobuf/src -I../../third_party/ced/src -I../../third_party/icu/source/common -I../../third_party/icu/source/i18n -I../../skia/config -I../../skia/ext -I../../third_party/skia/include/c -I../../third_party/skia/include/config -I../../third_party/skia/include/core -I../../third_party/skia/include/effects -I../../third_party/skia/include/encode -I../../third_party/skia/include/gpu -I../../third_party/skia/include/images -I../../third_party/skia/include/lazy -I../../third_party/skia/include/pathops -I../../third_party/skia/include/pdf -I../../third_party/skia/include/pipe -I../../third_party/skia/include/ports -I../../third_party/skia/include/utils -I../../third_party/skia/src/gpu -I../../third_party/skia/src/sksl -I../../third_party/libwebm/source -I../../third_party/protobuf/src -Igen/protoc_out -I../../third_party/leveldatabase -I../../third_party/leveldatabase/src -I../../third_party/leveldatabase/src/include -I../../third_party/webrtc_overrides -I../../testing/gtest/include -I../../third_party/webrtc -I../../third_party/webrtc_overrides -I../../third_party/webrtc -Igen/third_party/metrics_proto -I../../third_party/boringssl/src/include -I../../third_party/blink -Igen/third_party/blink -I../../v8/include -Igen/v8/include -I../../third_party/angle/src/common/third_party/base -Igen/angle -I../../third_party/brotli/include -I../../third_party/re2/src -I../../third_party/zlib -I../../third_party/mesa/src/include -I../../third_party/expat/files/lib /utf-8 /X -fcolor-diagnostics -Xclang -mllvm -Xclang -instcombine-lower-dbg-declare=0 /Gy /FS /bigobj /d2FastFail /Zc:sizedDealloc- -fmsc-version=1911 -m64 /W4 -Wimplicit-fallthrough -Wthread-safety /wd4091 /wd4127 /wd4251 /wd4275 /wd4312 /wd4324 /wd4351 /wd4355 /wd4503 /wd4589 /wd4611 /wd4100 /wd4121 /wd4244 /wd4505 /wd4510 /wd4512 /wd4610 /wd4838 /wd4995 /wd4996 /wd4456 /wd4457 /wd4458 /wd4459 /wd4702 -Wno-missing-field-initializers -Wno-unused-parameter -Wno-c++11-narrowing -Wno-covered-switch-default -Wno-unneeded-internal-declaration -Wno-inconsistent-missing-override -Wno-undefined-var-template -Wno-nonportable-include-path -Wno-address-of-packed-member -Wno-unused-lambda-capture -Wno-user-defined-warnings -Wno-enum-compare-switch -Wno-null-pointer-arithmetic /O1 /Ob2 /Oy- /Zc:inline /Gw /Oi /MT -Wheader-hygiene -Wstring-conversion -Wtautological-overlap-compare -Wshadow /wd4267 /TP /wd4577 /GR- /c gen/content/browser/browser_jumbo_11.cc /Foobj/content/browser/browser/browser_jumbo_11.obj /Fd"obj/content/browser/browser_cc.pdb"
clang-cl.exe: warning: argument unused during compilation: '/X' [-Wunused-command-line-argument]
In file included from gen/content/browser/browser_jumbo_11.cc:51:
.\../../content/browser/renderer_host/render_process_host_impl.cc(2850,15):  error: no member named 'kDisableWebRtcHWDecoding' in namespace 'switches'
    switches::kDisableWebRtcHWDecoding,
    ~~~~~~~~~~^
.\../../content/browser/renderer_host/render_process_host_impl.cc(2851,15):  error: no member named 'kDisableWebRtcHWEncoding' in namespace 'switches'; did you mean 'kDisableWebRtcEncryption'?
    switches::kDisableWebRtcHWEncoding,
    ~~~~~~~~~~^~~~~~~~~~~~~~~~~~~~~~~~
              kDisableWebRtcEncryption
../..\content/public/common/content_switches.h(251,34):  note: 'kDisableWebRtcEncryption' declared here
CONTENT_EXPORT extern const char kDisableWebRtcEncryption[];
                                 ^
In file included from gen/content/browser/browser_jumbo_11.cc:51:
.\../../content/browser/renderer_host/render_process_host_impl.cc(2886,34):  error: no matching function for call to 'ArraySizeHelper'
                                 arraysize(kSwitchNames));
                                 ^~~~~~~~~~~~~~~~~~~~~~~
../..\base/macros.h(54,34):  note: expanded from macro 'arraysize'
#define arraysize(array) (sizeof(ArraySizeHelper(array)))
                                 ^~~~~~~~~~~~~~~
../..\base/macros.h(53,40):  note: candidate template ignored: could not match 'T [N]' against 'const char *const []'
template <typename T, size_t N> char (&ArraySizeHelper(T (&array)[N]))[N];
                                       ^
3 errors generated.
[17511/21319] CXX obj/content/browser/browser/browser_jumbo_10.obj
clang-cl.exe: warning: argument unused during compilation: '/X' [-Wunused-command-line-argument]
[17512/21319] CXX obj/content/browser/browser/browser_jumbo_9.obj
clang-cl.exe: warning: argument unused during compilation: '/X' [-Wunused-command-line-argument]
[17513/21319] CXX obj/content/browser/browser/browser_jumbo_6.obj
FAILED: obj/content/browser/browser/browser_jumbo_6.obj
../../third_party/llvm-build/Release+Asserts/bin/clang-cl.exe /nologo /showIncludes  "-imsvcC:\Program Files (x86)\Microsoft Visual Studio\2017\Community\VC\Tools\MSVC\14.12.25827\ATLMFC\include" "-imsvcC:\Program Files (x86)\Microsoft Visual Studio\2017\Community\VC\Tools\MSVC\14.12.25827\include" "-imsvcC:\Program Files (x86)\Windows Kits\NETFXSDK\4.6.1\include\um" "-imsvcC:\Program Files (x86)\Windows Kits\10\include\10.0.15063.0\ucrt" "-imsvcC:\Program Files (x86)\Windows Kits\10\include\10.0.15063.0\shared" "-imsvcC:\Program Files (x86)\Windows Kits\10\include\10.0.15063.0\um" "-imsvcC:\Program Files (x86)\Windows Kits\10\include\10.0.15063.0\winrt" "-imsvcC:\Program Files (x86)\Microsoft Visual Studio\2017\Community\VC\Tools\MSVC\14.12.25827\ATLMFC\include" "-imsvcC:\Program Files (x86)\Microsoft Visual Studio\2017\Community\VC\Tools\MSVC\14.12.25827\include" "-imsvcC:\Program Files (x86)\Windows Kits\NETFXSDK\4.6.1\include\um" "-imsvcC:\Program Files (x86)\Windows Kits\10\include\10.0.16299.0\ucrt" "-imsvcC:\Program Files (x86)\Windows Kits\10\include\10.0.16299.0\shared" "-imsvcC:\Program Files (x86)\Windows Kits\10\include\10.0.16299.0\um" "-imsvcC:\Program Files (x86)\Windows Kits\10\include\10.0.16299.0\winrt" -DENABLE_SCREEN_CAPTURE=1 -D__ATLHOST_H__ -DV8_DEPRECATION_WARNINGS -DUSE_AURA=1 -DNO_TCMALLOC -DOFFICIAL_BUILD -DCHROMIUM_BUILD -D"CR_CLANG_REVISION=\"328716-2\"" -D_HAS_EXCEPTIONS=0 -D__STD_C -D_CRT_RAND_S -D_CRT_SECURE_NO_DEPRECATE -D_SCL_SECURE_NO_DEPRECATE -DDEPRECATEDENUMERATOR(x)=[[deprecated(x)]] -D_ATL_NO_OPENGL -D_WINDOWS -DCERT_CHAIN_PARA_HAS_EXTRA_FIELDS -DPSAPI_VERSION=1 -DWIN32 -D_SECURE_ATL -D_USING_V110_SDK71_ -DWINAPI_FAMILY=WINAPI_FAMILY_DESKTOP_APP -DWIN32_LEAN_AND_MEAN -DNOMINMAX -D_UNICODE -DUNICODE -DNTDDI_VERSION=0x0A000002 -D_WIN32_WINNT=0x0A00 -DWINVER=0x0A00 -DNDEBUG -DNVALGRIND -DDYNAMIC_ANNOTATIONS_ENABLED=0 -DCONTENT_IMPLEMENTATION -DV8_USE_EXTERNAL_STARTUP_DATA -DWEBP_EXTERN=extern -DUSE_EGL -DGOOGLE_PROTOBUF_NO_RTTI -DGOOGLE_PROTOBUF_NO_STATIC_INITIALIZER -DU_USING_ICU_NAMESPACE=0 -DU_ENABLE_DYLOAD=0 -DU_STATIC_IMPLEMENTATION -DICU_UTIL_DATA_IMPL=ICU_UTIL_DATA_FILE -DUCHAR_TYPE=wchar_t -DSK_IGNORE_LINEONLY_AA_CONVEX_PATH_OPTS -DSK_HAS_PNG_LIBRARY -DSK_HAS_WEBP_LIBRARY -DSK_HAS_JPEG_LIBRARY -DSK_SUPPORT_GPU=1 -DGR_GL_FUNCTION_TYPE=__stdcall -DLEVELDB_PLATFORM_CHROMIUM=1 -DDeleteFile=DeleteFileW -DOS_WIN -DWEBRTC_NON_STATIC_TRACE_EVENT_HANDLERS=0 -DGTEST_RELATIVE_PATH -DWEBRTC_CHROMIUM_BUILD -DWEBRTC_WIN -D_CRT_SECURE_NO_WARNINGS -DMESA_EGL_NO_X11_HEADERS -DXML_STATIC -DNO_MAIN_THREAD_WRAPPING -DFLAC__NO_DLL -I. -I../.. -Igen -I../../third_party/libwebp/src -I../../third_party/khronos -I../../gpu -I../../third_party/wtl/include -I../../third_party/libyuv/include -I../../third_party/protobuf/src -I../../third_party/ced/src -I../../third_party/icu/source/common -I../../third_party/icu/source/i18n -I../../skia/config -I../../skia/ext -I../../third_party/skia/include/c -I../../third_party/skia/include/config -I../../third_party/skia/include/core -I../../third_party/skia/include/effects -I../../third_party/skia/include/encode -I../../third_party/skia/include/gpu -I../../third_party/skia/include/images -I../../third_party/skia/include/lazy -I../../third_party/skia/include/pathops -I../../third_party/skia/include/pdf -I../../third_party/skia/include/pipe -I../../third_party/skia/include/ports -I../../third_party/skia/include/utils -I../../third_party/skia/src/gpu -I../../third_party/skia/src/sksl -I../../third_party/libwebm/source -I../../third_party/protobuf/src -Igen/protoc_out -I../../third_party/leveldatabase -I../../third_party/leveldatabase/src -I../../third_party/leveldatabase/src/include -I../../third_party/webrtc_overrides -I../../testing/gtest/include -I../../third_party/webrtc -I../../third_party/webrtc_overrides -I../../third_party/webrtc -Igen/third_party/metrics_proto -I../../third_party/boringssl/src/include -I../../third_party/blink -Igen/third_party/blink -I../../v8/include -Igen/v8/include -I../../third_party/angle/src/common/third_party/base -Igen/angle -I../../third_party/brotli/include -I../../third_party/re2/src -I../../third_party/zlib -I../../third_party/mesa/src/include -I../../third_party/expat/files/lib /utf-8 /X -fcolor-diagnostics -Xclang -mllvm -Xclang -instcombine-lower-dbg-declare=0 /Gy /FS /bigobj /d2FastFail /Zc:sizedDealloc- -fmsc-version=1911 -m64 /W4 -Wimplicit-fallthrough -Wthread-safety /wd4091 /wd4127 /wd4251 /wd4275 /wd4312 /wd4324 /wd4351 /wd4355 /wd4503 /wd4589 /wd4611 /wd4100 /wd4121 /wd4244 /wd4505 /wd4510 /wd4512 /wd4610 /wd4838 /wd4995 /wd4996 /wd4456 /wd4457 /wd4458 /wd4459 /wd4702 -Wno-missing-field-initializers -Wno-unused-parameter -Wno-c++11-narrowing -Wno-covered-switch-default -Wno-unneeded-internal-declaration -Wno-inconsistent-missing-override -Wno-undefined-var-template -Wno-nonportable-include-path -Wno-address-of-packed-member -Wno-unused-lambda-capture -Wno-user-defined-warnings -Wno-enum-compare-switch -Wno-null-pointer-arithmetic /O1 /Ob2 /Oy- /Zc:inline /Gw /Oi /MT -Wheader-hygiene -Wstring-conversion -Wtautological-overlap-compare -Wshadow /wd4267 /TP /wd4577 /GR- /c gen/content/browser/browser_jumbo_6.cc /Foobj/content/browser/browser/browser_jumbo_6.obj /Fd"obj/content/browser/browser_cc.pdb"
clang-cl.exe: warning: argument unused during compilation: '/X' [-Wunused-command-line-argument]
In file included from gen/content/browser/browser_jumbo_6.cc:43:
.\../../content/browser/gpu/gpu_process_host.cc(138,15):  error: no member named 'kDisableWebRtcHWEncoding' in namespace 'switches'; did you mean 'kDisableWebRtcEncryption'?
    switches::kDisableWebRtcHWEncoding,
    ~~~~~~~~~~^~~~~~~~~~~~~~~~~~~~~~~~
              kDisableWebRtcEncryption
../..\content/public/common/content_switches.h(251,34):  note: 'kDisableWebRtcEncryption' declared here
CONTENT_EXPORT extern const char kDisableWebRtcEncryption[];
                                 ^
1 error generated.
[17514/21319] CXX obj/content/browser/browser/browser_jumbo_15.obj
clang-cl.exe: warning: argument unused during compilation: '/X' [-Wunused-command-line-argument]
[17515/21319] CXX obj/content/browser/browser/browser_jumbo_8.obj
clang-cl.exe: warning: argument unused during compilation: '/X' [-Wunused-command-line-argument]
[17516/21319] CXX obj/content/browser/browser/browser_jumbo_7.obj
clang-cl.exe: warning: argument unused during compilation: '/X' [-Wunused-command-line-argument]
[17517/21319] CXX obj/content/browser/browser/browser_jumbo_5.obj
clang-cl.exe: warning: argument unused during compilation: '/X' [-Wunused-command-line-argument]
[17518/21319] CXX obj/content/browser/browser/browser_jumbo_4.obj
clang-cl.exe: warning: argument unused during compilation: '/X' [-Wunused-command-line-argument]
[17519/21319] CXX obj/content/browser/browser/browser_jumbo_12.obj
clang-cl.exe: warning: argument unused during compilation: '/X' [-Wunused-command-line-argument]
ninja: build stopped: subcommand failed.
```

**Comment by dubvulture** [comment]
Cannot buil linux_portable on Ubuntu 16.04

```
./out/Default/gn gen out/Default --args="host_cpu=\"x64\" blink_symbol_level=0 clang_use_chrome_plugins=false enable_ac3_eac3_audio_demuxing=true enable_google_now=false enable_hangout_services_extension=false enable_hevc_demuxing=true enable_iterator_debugging=false enable_mdns=false enable_mse_mpeg2ts_stream_parser=true enable_nacl=false enable_nacl_nonsfi=false enable_one_click_signin=false enable_reading_list=false enable_remoting=false enable_reporting=false enable_service_discovery=false enable_swiftshader=false enable_widevine=true exclude_unwind_tables=true fatal_linker_warnings=false ffmpeg_branding=\"Chrome\" fieldtrial_testing_like_official_build=true google_api_key=\"\" google_default_client_id=\"\" google_default_client_secret=\"\" is_clang=true is_debug=false is_official_build=true optimize_webui=false proprietary_codecs=true safe_browsing_mode=0 symbol_level=0 treat_warnings_as_errors=false use_gnome_keyring=false use_jumbo_build=true use_official_google_api_keys=false use_ozone=false use_sysroot=false use_unofficial_version_number=false custom_toolchain=\"//build/toolchain/linux/unbundle:default\" host_toolchain=\"//build/toolchain/linux/unbundle:default\" linux_use_bundled_binutils=false optimize_for_size=false use_gtk3=true use_kerberos=false use_lld=true use_vaapi=true" --fail-on-unused-args
Done. Made 8488 targets from 1535 files in 6893ms
ninja -j1 -C out/Default chrome chrome_sandbox content_shell chromedriver
ninja: Entering directory `out/Default'
ninja: error: '../../third_party/test_fonts/test_fonts/Arimo-Bold.ttf', needed by 'test_fonts/Arimo-Bold.ttf', missing and no known rule to make it
debian/rules:120: recipe for target 'override_dh_auto_build-arch' failed
make[1]: *** [override_dh_auto_build-arch] Error 1
make[1]: Leaving directory '/home/manuel/ungoogled-chromium/buildspace/tree'
debian/rules:105: recipe for target 'binary' failed
make: *** [binary] Error 2
dpkg-buildpackage: error: fakeroot debian/rules binary gave error exit status 2
```

**Comment by Wyse-** [comment]
Build of the master branch on Windows is indeed failing. I pretty much got the same error as @Ti-R, here it is:
https://paste.debian.net/plain/1031758

**Comment by Eloston** [comment]
I have moved `67.0.3396.87-2` to prerelease status for now. To encourage more consistent testing, I am looking to switching my development model to include "prereleases" (same meaning as a GitHub prerelease). There will be no changes to the tagging scheme. I will work out the specifics of this system and document it somewhere.

@Ti-R @Wyse- e9ce6ca67ed84d68a2716e7cca8ba55d3feca963 should reverse the breaking changes I made.

@dubvulture I think it should be possible to just disable the dependency on the testing fonts.

**Comment by xsmile** [comment]
@dubvulture @Eloston `test_fonts` can most probably be skipped using the patch at  https://github.com/Eloston/ungoogled-chromium/commit/ba903b2e93036ba366ca7be5f9e90d39b839bb2e#diff-31eef377f22cab09ac4b8eabe94dd6b2 which was removed a while ago.

**Comment by Wyse-** [comment]
@Eloston, @Ti-R build successful on Windows (commit e9ce6ca67ed84d68a2716e7cca8ba55d3feca963 of develop branch). Compiled binaries available [here](https://github.com/Wyse-/ungoogled-chromium/releases/tag/67.0.3396.87-3).

**Comment by Eloston** [comment]
@seppiofish I've never heard of this issue. Do you know what file(s) are growing in size?

**Comment by seppiofish** [comment]
@Eloston yes, a file named `BrowserMetrics-spare.pma` of 8MB is created and then a folder named `BrowserMetrics`, in which new files `BrowsersMetrics-<hex_string>.pma` of 8Mb of size each are constantly created growing the profile size indefinitely.

**Comment by xsmile** [comment]
@seppiofish I can confirm the issue. My profile directory contains about 10GB consisting of many 8MB sized `BrowserMetrics-*.pma` files. A new file seems to be created each time a new browser session is started.
Starting with version 67 `inox-patchset/9000-disable-metrics.patch` was changed to use Chromium's new `DummyHistogram` class to prevent histogram recording and apparently this has some unwanted side-effects. I will look into it for the next stable release in two weeks.
In the meantime you should disable write access to the `BrowserMetrics` directory to prevent .pma files from being written. I'm not sure how psd will react to this though.

**Comment by Ti-R** [comment]
@Eloston, @Wyse- build successful on Windows also for me.

Took me a while to come back.

Got some problems some days agos with 
> The procedure entry point _o_memset could not be located in the dynamic link library api-mswin-crt-private-l1-1-0.dll

Due to some of my Windows Software Development Kit missing "Debugging Tools for Windows" option
All is ok now.

I compile this version
* Move VA-API patch back into Linux bundles
e9ce6ca67ed84d68a2716e7cca8ba55d3feca963

**Comment by Eloston** [comment]
@Wyse- If you believe the Windows binaries are good, you can submit them to the contributor binaries site (ungoogled-chromium-binaries)

**Comment by Eloston** [comment]
@Wyse- Hmm, you brought up a good point about master and develop branches. It's unnecessary to have two branches like this with the new release model that I've developed. I'll have to consolidate them.

I also don't really have a system to receive feedback for testing, to promote development binaries to release binaries.

Some documentation about the new release system is here (I will need to organize it a bit better):
* https://github.com/ungoogled-software/ungoogled-chromium-binaries#changing-version-statuses
* https://ungoogled-software.github.io/ungoogled-chromium-wiki/statuses

**Comment by Eloston** [comment]
Closing due to development on version 68.
